content/it/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,126 @@
+---
+title: I componenti di Kubernetes
+content_template: templates/concept
+weight: 20
+card: 
+  name: concepts
+  weight: 20
+---
+
+{{% capture overview %}}
+Facendo il deployment di Kubernetes, ottieni un cluster.
+{{< glossary_definition term_id="cluster" length="all" prepend="Un cluster è">}}
+
+Questo documento describe i diversi componenti che sono necessari per avere 
+un cluster Kubernetes completo e funzionante.
+
+Questo è un diagramma di un cluster Kubernetes con tutti i componenti e le loro relazioni.
+
+![Components of Kubernetes](/images/docs/components-of-kubernetes.png)
+
+{{% /capture %}}
+
+{{% capture body %}}
+## Componenti del Master
+
+I componenti del Master forniscono la control plane del cluster.
+I componenti del Master fanno decisioni globali sul cluster (ad esempio, lo scheduling), e individuano e rispondono a eventi derivanti dal cluster (ad esempio, l'avvio di un nuovo {{< glossary_tooltip text="pod" term_id="pod">}} quando il valore `replicas` di un deployment non è soddisfatto).
+
+I componenti del Master possono essere eseguiti su qualsiasi nodo del cluster.
+Solitamente, per motivi di facilità, gli script di installazione tendono a eseguire tutti i componenti del Master sulla stessa macchina, senza l'utilizzo di container.
+Vedi [creare un cluster in High-Availability](/docs/admin/high-availability/) per un esempio di un'installazione multi-master.
+
+### kube-apiserver
+
+{{< glossary_definition term_id="kube-apiserver" length="all" >}}
+
+### etcd
+
+{{< glossary_definition term_id="etcd" length="all" >}}
+
+### kube-scheduler
+
+{{< glossary_definition term_id="kube-scheduler" length="all" >}}
+
+### kube-controller-manager
+
+{{< glossary_definition term_id="kube-controller-manager" length="all" >}}
+
+Questi controller includono:
+
+  * Node Controller: Responsabile dell'identificazione e della risposta ad eventi in cui i nodi diventano non disponibili.
+  * Replication Controller: Responsabile per il mantenimento del corretto numero di pod per ogni replication controller del sistema.
+  * Endpoints Controller: Popola gli oggetti Endpoints (cioè, mette in relazioni i Pods con i Services).
+  * Service Account & Token Controllers: Creano gli account di default e i token di accesso alle API per i nuovi namespaces.
+
+### cloud-controller-manager
+
+Il [cloud-controller-manager](/docs/tasks/administer-cluster/running-cloud-controller/) esegui i controller che interagiscono con i cloud provider sottostanti.
+Il cloud-controller-manager è una funzionalità alpha introdotta in Kubernetes 1.6.
```

Comment:
I'm pretty sure this is not anymore true, so my suggestion is to remove this sentence.

Also, the following sentences in this paragraph are out of data, and IMO, too specific for an Kubernetes overview. So my proposal is to remove them too

Reply:
If the English localization is wrong, the norm is to fix that upstream (see #18086)